nano/core_test/toml.cpp

```diff
@@ -76,24 +76,162 @@ TEST (toml, daemon_config_update_array)
 	ASSERT_EQ (c.node.preconfigured_peers[0], "test-peer.org");
 }
 
+/** Empty rpc config file should match a default config object */
+TEST (toml, rpc_config_deserialize_defaults)
+{
+	std::stringstream ss;
+
+	// A config file with values that differs from test-net defaults
+	ss << R"toml(
+	[process]
+	)toml";
+
+	nano::tomlconfig t;
+	t.read (ss);
+	nano::rpc_config conf;
+	nano::rpc_config defaults;
+	conf.deserialize_toml (t);
+
+	ASSERT_FALSE (t.get_error ()) << t.get_error ().get_message ();
+
+	ASSERT_EQ (conf.address, defaults.address);
+	ASSERT_EQ (conf.enable_control, defaults.enable_control);
+	ASSERT_EQ (conf.max_json_depth, defaults.max_json_depth);
+	ASSERT_EQ (conf.max_request_size, defaults.max_request_size);
+	ASSERT_EQ (conf.port, defaults.port);
+
+	ASSERT_EQ (conf.rpc_process.io_threads, defaults.rpc_process.io_threads);
+	ASSERT_EQ (conf.rpc_process.ipc_address, defaults.rpc_process.ipc_address);
+	ASSERT_EQ (conf.rpc_process.ipc_port, defaults.rpc_process.ipc_port);
+	ASSERT_EQ (conf.rpc_process.num_ipc_connections, defaults.rpc_process.num_ipc_connections);
+}
+
 /** Empty config file should match a default config object */
 TEST (toml, daemon_config_deserialize_defaults)
```

Comment:
I think `conf_height_processor_batch_min_time` is missing

Reply:
yeah https://github.com/nanocurrency/nano-node/pull/2105 was missed wrt toml.

Reply:
#2105 was missed